src/main/asciidoc/_chapters/ops_mgt.adoc

```diff
@@ -2745,6 +2745,40 @@ A snapshot is only a representation of a table during a window of time.
 The amount of time the snapshot operation will take to reach each Region Server may vary from a few seconds to a minute, depending on the resource load and speed of the hardware or network, among other factors.
 There is also no way to know whether a given insert or update is in memory or has been flushed.
 
+
+.Take a Snapshot With TTL
+Snapshots have a lifecycle that is independent from the table from which they are created.
+Although data in a table may be stored with TTL the data files containing them become
+frozen by the snapshot. Space consumed by expired cells will not be reclaimed by normal
+table housekeeping like compaction. While this is expected it can be inconvenient at scale.
+When many snapshots are under management and the data in various tables is expired by
+TTL some notion of optional TTL (and optional default TTL) for snapshots could be useful.
+
+Default Snapshot TTL = 30 Days
+
+While creating a Snapshot, if TTL in seconds is not specified, the Default `TTL` will be applicable.
```

Comment:
I don't think using this TTL version snapshot to replace current one is a good idea.
Open to discussion about this more.

Reply:
Sorry, I didn't get you. Are you suggesting that we should not keep 30 days as default snapshot TTL? 
I am not sure what attribute you are referring to with: `replace current one`